docs/root/configuration/upstream/cluster_manager/cluster_circuit_breakers.rst

```diff
@@ -11,14 +11,14 @@ The following is an example circuit breaker configuration:
 .. code-block:: yaml
 
   circuit_breakers:
-  thresholds:
-    - priority: "DEFAULT"
-      max_requests: 75
-      max_pending_requests: 35
-      retry_budget:
-        budget_percent:
-          value: 25.0
-        min_retry_concurrency: 10
+    thresholds:
+      - priority: "DEFAULT"
+        max_requests: 75
+        max_pending_requests: 35
+        retry_budget:
+          budget_percent:
+            value: 25.0
+          min_retry_concurrency: 10
```

Comment:
Super nit, sorry, for consistency, could you help to follow the indentation for "list item" like e.g. this one: https://www.envoyproxy.io/docs/envoy/v1.15.0/intro/arch_overview/security/ssl#example-configuration? While it is not written anywhere, I tend to see this is how we write yaml in envoy docs.

```yaml
    thresholds:
    - priority: "DEFAULT"
      max_requests: 75
      max_pending_requests: 35
      retry_budget:
        budget_percent:
          value: 25.0
        min_retry_concurrency: 10
```

Thanks, @ujihisa!

Reply:
Fixed! :+1: